.github/workflows/ci.canary.yml

```diff
@@ -256,20 +251,44 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPMJS_TOKEN: ${{ secrets.NPMJS_TOKEN }}
 
+  publish-sdk-html-docs:
+    name: Publish canary HTML docs
+
+    needs: [publish-npm-packages]
+
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
+        with:
+          repository: superfluid-finance/build-scripts
+          path: build-scripts
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: "yarn"
+          registry-url: https://registry.npmjs.org/
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build
+        run: |
+          yarn --cwd packages/sdk-core set-default-subgraph-release-tag
+          yarn --cwd packages/sdk-core generate-graphql-schema:dev
+          yarn build
+        env:
+          SUBGRAPH_RELEASE_TAG: dev
+
       - name: Build HTML documentation of SDK-s
         run: |
           yarn --cwd packages/sdk-core doc:html
           yarn --cwd packages/sdk-redux doc:html
 
-      - name: Get versions for sdk-core & sdk-redux
```

Comment:
this is not used anywhere - well `sdk-versions` is commented out in `handler.update-evm-contracts-docs.yml`